DP3TApp/Logic/Tracing/TracingManager.swift

```diff
@@ -42,53 +42,44 @@ class TracingManager: NSObject {
     }
 
     func initialize() {
-        do {
-            let bucketBaseUrl = Environment.current.configService.baseURL
-            let reportBaseUrl = Environment.current.publishService.baseURL
-            // JWT is not supported for now since the backend keeps rotating the private key
-
-            #if TEST_ENTITLEMENT
-                let descriptor = ApplicationDescriptor(appId: appId,
-                                                       bucketBaseUrl: bucketBaseUrl,
-                                                       reportBaseUrl: reportBaseUrl,
-                                                       jwtPublicKey: Environment.current.jwtPublicKey,
-                                                       mode: .test)
-            #else
-                let descriptor = ApplicationDescriptor(appId: appId,
-                                                       bucketBaseUrl: bucketBaseUrl,
-                                                       reportBaseUrl: reportBaseUrl,
-                                                       jwtPublicKey: Environment.current.jwtPublicKey)
-            #endif
+        let bucketBaseUrl = Environment.current.configService.baseURL
+        let reportBaseUrl = Environment.current.publishService.baseURL
+        // JWT is not supported for now since the backend keeps rotating the private key
```

Comment:
I think this comment stems from the pre-production phase and can be removed now a